bacon-vs-kefir-api.md

```diff
@@ -155,11 +155,11 @@
 | ----- | ----- | -------- |
 | `stream.map(property)` | Use `.sampledBy()` |  |
 | `obs.filter(property)` | `obs.filterBy(obs)` |  |
-| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Only property supported in Bacon? |
+| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Bacon supports property or predicate, Kefir supports any observable |
```

Comment:
There is versions of these methods with predicate in the "Modify an observable" section. I think we'd better add something like "see also `takeWhile(fn)` above", to not confuse anybody that Kefir doesn't have version with predicate.